assets/css/initial-setup.css

```diff
@@ -1 +1 @@
right:40px;font-size:15px;}@media (max-width:768px){#wpwrap .settings .nav>li>a{margin-right:10px}}#wpwrap .settings .nav>li>a:hover,#wpwrap .settings .nav>li>a:focus{text-decoration:none;background-color:#fff;border-color:transparent transparent #ddd}#wpwrap .settings .nav-tabs>li.active>a,#wpwrap .settings .nav-tabs>li.active>a:hover,#wpwrap .settings .nav-tabs>li.active>a:focus{color:#33b6e3;border-color:transparent transparent #33b6e3}#wpwrap .settings .form-wrap p,#wpwrap .settings p.description,#wpwrap .settings p.help,#wpwrap .settings span.description{font-style:normal;color:rgba(0,0,0,0.5)}#wpwrap .settings label{margin-right:10px}#wpwrap .settings .connections{margin-top:30px}#wpwrap .a0-table{margin:2em 2%;width:96%}#wpwrap #manuallySetToken{margin-right:15px}#wpwrap .a0-feedback{background-color:#f2f5f8;padding:30px;margin-top:30px;border-radius:3px;}#wpwrap .a0-feedback h2{font-size:18px;margin-bottom:15px;}#wpwrap .a0-feedback h2.message{margin:0}#wpwrap .a0-feedback .a0-separator{margin:20px 0}#wpwrap .a0-feedback input.feedback_calification{display:none}#wpwrap .a0-feedback input:checked + .feedback-face{opacity:1}#wpwrap .a0-feedback .a0-buttons{margin:15px 0 0;}#wpwrap .a0-feedback .a0-buttons .a0-button{min-width:auto;cursor:pointer}#wpwrap .a0-feedback textarea{padding:10px;height:90px}#wpwrap .a0-feedback .feedback-face{width:40px;height:40px;display:inline-block;background-size:contain;opacity:.4;margin-right:30px;cursor:pointer;}#wpwrap .a0-feedback .feedback-face:hover{opacity:.7}#wpwrap .a0-feedback .feedback-face.calification-1{background-image:url("../img/rates/01.svg")}#wpwrap .a0-feedback .feedback-face.calification-2{background-image:url("../img/rates/02.svg")}#wpwrap .a0-feedback .feedback-face.calification-3{background-image:url("../img/rates/03.svg")}#wpwrap .a0-feedback .feedback-face.calification-4{background-image:url("../img/rates/04.svg")}#wpwrap .a0-feedback .feedback-face.calification-5{background-image:url("../img/rates/05.svg")}@-moz-keyframes loaderAnim{0%{-webkit-transform:rotate(0);-moz-transform:rotate(0);-ms-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);-moz-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg)}}@-webkit-keyframes loaderAnim{0%{-webkit-transform:rotate(0);-moz-transform:rotate(0);-ms-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);-moz-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg)}}@-o-keyframes loaderAnim{0%{-webkit-transform:rotate(0);-moz-transform:rotate(0);-ms-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);-moz-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg)}}@keyframes loaderAnim{0%{-webkit-transform:rotate(0);-moz-transform:rotate(0);-ms-transform:rotate(0);transform:rotate(0)}100%{-webkit-transform:rotate(360deg);-moz-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg)}}
```

Comment:
I'm a little worried about what happened here. It's minified so hard to see what the difference is.

Reply:
I accepted the >>>>>>> changes

Reply:
Those actually came from your merge and indicate a conflict. Try copying the file in from `master` and pushing:

https://github.com/auth0/wp-auth0/blob/master/assets/css/initial-setup.css

Reply:
Sure, is now ok?